lemminx-liberty/src/main/java/io/openliberty/tools/langserver/lemminx/services/SettingsService.java

```diff
@@ -56,6 +56,17 @@ public String getLibertyVersion() {
     return null;
   }
 
+  public String getLibertyRuntime() {
+    if (settings != null) {
+      String runtime = settings.getRuntime();
+      if (runtime != null) {
+        return runtime;
+      }
+    }
+
+    return null;
```

Comment:
This method seems inefficient. I think it can be reduced to the following.
```
return settings != null ? settings.getRuntime() : null;
```

Reply:
Fixed.